OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -41,14 +41,18 @@ public void Killed(Actor self, AttackInfo e)
 
 			var valued = self.Info.TraitInfoOrDefault<ValuedInfo>();
 
-			// Default experience is 100 times our value
 			var exp = info.Experience >= 0
 				? info.Experience
-				: valued != null ? valued.Cost * 100 : 0;
+				: valued != null ? valued.Cost : 0;
 
 			var killer = e.Attacker.TraitOrDefault<GainsExperience>();
 			if (killer != null)
 				killer.GiveExperience(exp);
+
+			if (info.Experience > 0)
+			{
```

Comment:
Note that our code style is to omit braces around single-line blocks.  Please remove them from here and the addition to PlayerStatistics.